sample/src/main/kotlin/com/chuckerteam/chucker/sample/MainActivity.kt

```diff
@@ -35,6 +39,21 @@ class MainActivity : AppCompatActivity() {
                 }
             }
 
+            doGraphql?.setOnClickListener {
+                for(task in graphQLTasks) {
+                    task.run("query GetCharacter( \$id: ID! ){\n" +
```

Comment:
is there a reason you pass this in here for the get request but hardcode it for the post request?